### ISSUE TITLE
 not reporting

### ISSUE THREAD

**Issue by KentPM** [wakatime/atom-wakatime#66]
despite appearing to function correctly the package is failing to report data from Atom.

**Comment by alanhamlett** [comment]
Can you try reinstalling the wakatime package inside Atom? We've had cases where the installation fails but reinstalling fixes the issue.

Also, do you have any error messages in your Atom Developer Console after turning on debug mode?

**Comment by KentPM** [comment]
I uninstalled and reinstalled several times with no apparent avail. In the console it does not throw any errors.

`[WakaTime] [DEBUG] [wakatime.coffee:36] Initializing WakaTime v6.0.10...
logger.coffee:36 [WakaTime] [DEBUG] [wakatime.coffee:269] downloading python...
nunjucks.coffee:1 loading nunjucks
logger.coffee:36 [WakaTime] [DEBUG] [wakatime.coffee:275] extracting python...
logger.coffee:36 [WakaTime] [DEBUG] [wakatime.coffee:280] Finished installing python.
logger.coffee:36 [WakaTime] [DEBUG] [wakatime.coffee:341] Downloading wakatime-cli...
logger.coffee:36 [WakaTime] [DEBUG] [wakatime.coffee:350] Extracting wakatime-master.zip file...
logger.coffee:36 [WakaTime] [DEBUG] [wakatime.coffee:90] Finished installing wakatime-cli.
logger.coffee:36 [WakaTime] [DEBUG] [wakatime.coffee:111] Finished initializing WakaTime.`

**Comment by alanhamlett** [comment]
All your DEBUG messages look correct. Do you have any errors in your `.wakatime.log` file? It should be located in your home folder `C:\Users\<user>\.wakatime.log`.

If there are no errors there, try reinstalling the package and make sure the whole folder at `C:\Users\<user>\.atom\packages\wakatime` is deleted when uninstalled.

**Comment by KentPM** [comment]
I looked for the wakatime.log, however it was nowwhere to be found. Uninstalled Atom completely deleting any lingering files. Reinstalled Atom with only wakatime with no other packages. I searched for the log file again, not found. Looked at the logger script and it appeared intact, so I wondered if python wasn't launching correctly. Launched the python app in the lib and  Windows threw an error for 

> api-ms-win-crt-runtime-l1-1-0.dll C runtime environment
for some reason windows update had missed that particular file.

*SOLUTION- to update the C runtime for Windows available at [windows support](https://support.microsoft.com/en-us/kb/2999226) for Win7-10.1.
